code/modules/crew_objectives/security_objectives.dm

```diff
@@ -0,0 +1,31 @@
+/*				SECURITY OBJECTIVES				*/
+
+/datum/objective/crew/enjoyyourstay
+	explanation_text = "Welcome aboard. Enjoy your stay."
+	jobs = "headofsecurity,securityofficer,warden,detective"
+	var/list/edglines = list("Welcome aboard. Enjoy your stay.", "You signed up for this.", "Abandon hope.", "The tide's gonna stop eventually.", "Hey, someone's gotta do it.", "No, you can't resign.", "Security is a mission, not an intermission.")
```

Comment:
Listen

This was removed over at Cit for good reason, the HoS committing suicide over the "abandon hope" edgline can be pretty amusing at times, but after a while ya gotta think if it's truly a good idea.

Reply:
It was indeed a bad idea, which is why I changed it from
`explanation_text = pick(edglines)`
to
`explanation_text = "Enforce Space Law to the best of your ability, and survive. [pick(edglines)]"`